src/Zir.zig

```diff
@@ -3416,8 +3416,9 @@ pub const DeclIterator = struct {
         it.extra_index += 5; // src_hash(4) + line(1)
         const name = it.zir.nullTerminatedString(it.zir.extra[it.extra_index]);
         it.extra_index += 3; // name(1) + value(1) + doc_comment(1)
-        it.extra_index += @as(u1, @truncate(flags >> 2));
-        it.extra_index += @as(u1, @truncate(flags >> 3));
+        it.extra_index += @as(u1, @truncate(flags >> 2)); // align
+        it.extra_index += @as(u1, @truncate(flags >> 3)); // link_section
+        it.extra_index += @as(u1, @truncate(flags >> 3)); // address_space
```

Comment:
Surely these should be using different bits?

Reply:
Ideally we should have separate bits, but right now we only have 4 bits for 5 attributes ( `pub`, `exported`, `align`, `linksection` and `addrspace`), so they have to share 1 bit as of now. Let's just adapt to the current implementation right now and redesign later?